### ISSUE TITLE
Story Promo tweaks

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#542]
**Is your feature request related to a problem? Please describe.**
After having the a11y swarm for this component. We noticed that some font colours applied are wrong and that the summary paragraph is missing the visibility property when is not displayed on mobile, to make sure that no assistive technology will see the content.

**Describe the solution you'd like**
- Replace `Headline` and `Link` colour from `C_SHADOW #3F3F42` to `C_EBON #222222`
- Add `visibility: hidden;` to the `Summary` under `600px`.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**
